src/BuildScriptGenerator/Golang/GolangBashBuildSnippet.sh.tpl

```diff
@@ -23,7 +23,11 @@ echo "   "
 
 # TODO: add support for nested dirs
 echo "building go app..."
-go build -o oryxBuildBinary
+doc="https://aka.ms/troubleshoot-go"
+suggestion="Please check your go.mod is valid. Try building locally first: go build"
```

Comment:
nit: I would slightly rephrase the last sentence here:

_Try building locally first with the following command: go build_

Reply:
I updated, thanks!